Util/immutables/gradle.properties

```diff
@@ -1,2 +1,2 @@
 io.deephaven.project.ProjectType=JAVA_LOCAL
-languageLevel=8
\ No newline at end of file
+deephaven.languageLevel=8
```

Comment:
prev line in this file suggests that if we want to make these more specific, let's namespace further? io.deepahaven.java.languageLevel perhaps?

Or drop `io.` prefix for the other line? I just don't want these files getting as bad as our prop files...